modules/oe_theme_content_project/oe_theme_content_project.module

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Project content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_project_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  // Run only on project content type.
+  if ($entity->bundle() !== 'oe_project') {
+    return;
+  }
+
+  // Field labels are different in frontend and backend.
+  // Here we override them, per view mode.
+  if ($build['#view_mode'] === 'full') {
+    $label_overrides = [
```

Comment:
Could we move this just above the foreach below? :smile:

Reply:
Has it been fixed ? I do not understand your idea

Reply:
It is correct order - default values are defined first, then we override one of them.